lms/djangoapps/courseware/tests/test_video_mongo.py

```diff
@@ -627,10 +631,23 @@ def test_get_html_with_mocked_edx_video_id(self):
         })
 
         self.assertEqual(
-            context,
-            self.item_descriptor.xmodule_runtime.render_template('video.html', expected_context)
+            self.get_context_dict_from_string(context),
+            self.get_context_dict_from_string(
+                self.item_descriptor.xmodule_runtime.render_template('video.html', expected_context)
+            )
         )
 
+    def get_context_dict_from_string(self, data):
+        """
+        Retrieve dictionary from string.
+        """
+        # Replace tuple and un-necessary info from inside string and get the dictionary.
+        cleaned_data = eval(data.split('((\'video.html\',')[1].replace("),\n {})", '').strip())  # pylint: disable=unicode-format-string
```

Comment:
It looks like literal_eval is the safer variant @awais786 

https://nedbatchelder.com/blog/201206/eval_really_is_dangerous.html  Let's change back to that and I think it should be fine.